.github/workflows/test-mitosheet-python.yml

```diff
@@ -54,3 +54,4 @@ jobs:
         PYTEST_SNOWFLAKE_USERNAME: ${{ secrets.PYTEST_SNOWFLAKE_USERNAME }}
         PYTEST_SNOWFLAKE_PASSWORD: ${{ secrets.PYTEST_SNOWFLAKE_PASSWORD }}
         PYTEST_SNOWFLAKE_ACCOUNT: ${{ secrets.PYTEST_SNOWFLAKE_ACCOUNT }}
+        OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
```

Comment:
Note that this only tests the case when the `OPENAI_API_KEY` is set. So we don't have any tests running in our CI when its not set

Reply:
Since we don't conditionally install a dependency like snowflake, I'm fine with this!